internal/service/folders.go

```diff
@@ -240,7 +240,9 @@ func (s *DashNGoImpl) UploadFolders(filter filters.Filter) []string {
 				continue
 			}
 		}
+
 		var newFolder models.CreateFolderCommand
```

Comment:
This is the same as the following line unless I'm missing something.

Reply:
I'm not sure I follow, this declares a new variable that will be used below.